docs/explanation/e-hardening.md

```diff
@@ -0,0 +1,119 @@
+| Azure     | [Azure security best practices and patterns](https://learn.microsoft.com/en-us/azure/security/fundamentals/best-practices-and-patterns), [Managed identities for Azure resource](https://learn.microsoft.com/en-us/entra/identity/managed-identities-azure-resources/)                                                                            |
+
+### Juju 
+
+Juju is the component responsible for orchestrating the entire lifecycle, from deployment to Day 2 operations, of 
+all applications. Therefore, it is imperative that it is set up securely. Please refer to the Juju documentation for more information on:
+
+* [Juju security](https://discourse.charmhub.io/t/juju-security/15684)
+* [How to harder your deployment](https://juju.is/docs/juju/harden-your-deployment)
+
+#### Cloud credentials
+
+When configuring the cloud credentials to be used with Juju, ensure that the users have correct permissions to operate at the required level. 
+Juju superusers responsible for bootstrapping and managing controllers require elevated permissions to manage several kind of resources, such as
+virtual machines, networks, storages, etc. Please refer to the references below for more information on the policies required to be used depending on the cloud. 
+
+| Cloud     | Cloud user policies                                                                                                                                                                                                                            |
+|-----------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| OpenStack |                                                                                                                                                                                                                                                |
+| AWS       | [Juju AWS Permission](https://discourse.charmhub.io/t/juju-aws-permissions/5307), [AWS Instance Profiles](https://discourse.charmhub.io/t/using-aws-instance-profiles-with-juju-2-9/5185), [Juju on AWS](https://juju.is/docs/juju/amazon-ec2) | 
+| Azure     | [Juju Azure Permission](https://juju.is/docs/juju/microsoft-azure), [How to use Juju with Microsoft Azure](https://discourse.charmhub.io/t/how-to-use-juju-with-microsoft-azure/15219)                                                                                                  |
+
+#### Juju users
+
+It is very important that the different juju users are set up with minimal permission depending on the scope for their operations. 
```

Comment:
Note for future plans: We have a task in our backlog to create a page about Users in the Explanation section. It will be nice to add such a page with more specific information and then link to it here.

Reply:
Totally! I agree!